vstools/enums/other.py

```diff
@@ -61,78 +60,36 @@ class Dar(Fraction):
     For more information, see <https://en.wikipedia.org/wiki/Display_aspect_ratio>.
     """
 
-    @overload
     @classmethod
-    def from_size(
-        cls,
-        width: int,
-        height: int,
-        sar: Sar | bool = True, /,
-        func: FuncExceptT | None = ...
-    ) -> Self:
+    def from_res(cls, width: int, height: int, sar: Sar | None = None) -> Dar:
         """
-        Get the Display Aspect Ratio from the clip's dimensions or Sample Aspect Ratio (SAR).
+        Get the DAR from the specified dimensions and SAR.
```

Comment:
I don't see why we're removing the meaning of the acronym here. Just makes it less understandable and harder to google about if you want to figure out what this does.